### PR TITLE
SD-3054 - add Http Push type of delivery to be used by Content API subscriber

### PR DESCRIPTION
**Pull request by mugurrus**

### CODE REVIEW

**Comment by amagdas** [comment]
:+1:

File: content_api/features/publish.feature [hunk 1]
```diff
@@ -45,7 +45,7 @@ Feature: Publish service
             "language": "en",
             "mimetype": "text/plain",
             "pubstatus": "usable",
-            "versions": "1"
+            "versions": 1
```
Comment (sivakuna-aap) [inline content_api/features/publish.feature:48]: Isn't it supposed to be `version` without (s)?

**Comment by sivakuna-aap** [comment]
Correct me if I am wrong - from the PR what I understood is the HTTP Push Type/Content API is dependent on the `version` property. But in reality when user publishes we are not incrementing or deal with `version` property. Irrespective of number of publish actions on an item the `version` value will always be 1. If some functionality is dependent on this property then we need to change the publish functionality as well.

**Comment by mugurrus** [comment]
On formatting to ninjs version takes the _current_version value:
https://github.com/superdesk/superdesk/blob/master/server/apps/publish/formatters/ninjs_formatter.py#L33

File: content_api/features/publish.feature (hunk 1 above)
Comment (mugurrus) [inline content_api/features/publish.feature:48]: Check the lines below, this test checks invalid fields detection.

Comment (sivakuna-aap) [inline content_api/features/publish.feature:48]: Thanks for the clarification. My bad I didn't check the whole file

**Comment by sivakuna-aap** [comment]
Yes, you were right. :+1:

**Pull request merged**